Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -2000,6 +2000,51 @@ public async Task FindSpecificPeerAsyncDepthFail()
             }
         }
 
+        [Fact(Timeout = Timeout)]
+        public async Task DoNotRetryFillWhenGetAllBlocksFromSender()
+        {
+            Swarm<DumbAction> receiver = CreateSwarm();
+            Swarm<DumbAction> sender = CreateSwarm();
+            await StartAsync(receiver);
+            await StartAsync(sender);
+
+            Block<DumbAction> b1 =
+                TestUtils.MineNext(receiver.BlockChain.Genesis, difficulty: 1024);
+            sender.BlockChain.Append(b1);
+
+            for (int i = 0; i < 1000; i++)
+            {
+                sender.BlockChain.Append(
+                    TestUtils.MineNext(sender.BlockChain.Tip, difficulty: 1024));
+            }
+
+            Log.Debug("Sender's BlockChain Tip index: #{index}", sender.BlockChain.Tip.Index);
+
+            try
+            {
+                await BootstrapAsync(sender, receiver.AsPeer);
+
+                sender.BroadcastBlock(b1);
+
+                await receiver.BlockReceived.WaitAsync();
+                Assert.Equal(
+                    2,
```

Comment:
Does it mean we can't receive all blocks(1,000) by one broadcasting? if then, it means that we changed the previous spec. 😕

Reply:
It is fixed at https://github.com/planetarium/libplanet/pull/1115/commits/1a4594dab6f013704f1998b250e3fce226a3b2fe